### DISCUSSION TITLE
Styled Component

### DISCUSSION THREAD

**Discussion by 14KGun** [sparcs-kaist/taxi-front#221] [Ideas]
## style 관련 논의
  Styled Component가 구지 필요할까요??
  front style 통합 리펙토링 관련해서 Styled Component 대안이 없을까요??
  
@imYourChoi, @GrasshopperBears, @UrWrstNightmare

**Comment by imYourChoi** [comment]
공통으로 쓰이는 style들은 MUI의 makestyle을 써서 별개의 파일에서 스타일 관리하는게 좋을 것 같습니다. 그렇게 하면 다크 테마와 기본 테마도 분리해서 관리하기도 편할것 같구요. 개인적으로 style component를 별로 안좋아하기도 합니다ㅎ
